### PR TITLE
blog: October newsletter

### PR DESCRIPTION
**Pull request by GregSutcliffe**
As always, more content is welcome (although it's pretty big already!).

### CODE REVIEW

**Comment by GregSutcliffe** [comment]
[x] ~~Got a last minute addition of a Deep Dive to add, don't merge until I've done that.~~

Done

**Comment by shlomizadok** [comment]
@GregSutcliffe++ that's an awesome write-up

File: _posts/2016-10-27-foreman-community-newsletter---october-2016.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+[mailing list][userslist] - please do join in the discussion if this is of
+interest to you, or you have experience to share!
+
+Note that installation of EL6 hosts from Foreman is still supported - it's the
+packages for Foreman itself that will no longer be produced for EL6.
+
+[el6]: https://groups.google.com/d/topic/foreman-users/lec9OBPJEGo/discussion
+[userslist]: https://groups.google.com/forum/#!forum/foreman-users
+
+### Katello 3.2 RC2
```
Comment (ehelms) [inline _posts/2016-10-27-foreman-community-newsletter---october-2016.md]: RC3 is going out today with the EL6 support if you want to update this :)

Comment (GregSutcliffe) [reply _posts/2016-10-27-foreman-community-newsletter---october-2016.md]: This is unlikely to be merged today anyway, I'll update tomorrow morning with a link to the announce. Thanks!

Comment (ehelms) [reply _posts/2016-10-27-foreman-community-newsletter---october-2016.md]: Thanks!

**Comment by ehelms** [comment]
You may want to throw in https://theforeman.org/2016/10/puppet4-integration.html to the blog section.

File: _posts/2016-10-27-foreman-community-newsletter---october-2016.md (hunk 1 above)
Comment (GregSutcliffe) [reply _posts/2016-10-27-foreman-community-newsletter---october-2016.md]: Done!

**Comment by GregSutcliffe** [comment]
@ehelms Both added (the blog is questionable since it'll be on the same page and RSS as this newsletter... but I added it anyway :P). Also added @mclueppers blogs on HA too.

**Comment by mmoll** [comment]
merged with small link fixes, thanks @GregSutcliffe!

**Pull request closed without merging**